app/waf/tslint.json

```diff
@@ -1,4 +1,7 @@
 {
   "$schema": "http://json.schemastore.org/tslint",
-  "extends": ["./node_modules/@loopback/build/config/tslint.common.json"]
+  "extends": ["./node_modules/@loopback/build/config/tslint.common.json"],
+  "rules": {
```

Comment:
What does this rule mean?
Let ts ignore the `invalid this`?

Reply:
We need to call this.timeout() to adjust UT timeout value, but a lint warning is not to use 'this' outside a class body. It disable this lint rule.

Reply:
No need, change it back, and remove the sleep(3000).

Reply:
Close this thread, since it is duplicated.